### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Tuesday, 11th of July 2023 17:30 CEST / 11:30am EDT / 8:30am PDT

### DISCUSSION THREAD

**Discussion by t-lo** [flatcar/Flatcar#1078] [Flatcar Office Hours]
- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording: https://www.youtube.com/watch?v=tgm1Rza_9e8
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
  
## Welcome
- Brief intro/check-in of the new participants joining the call. Please introduce yourself and share what brings you here today.

## Agenda items
- @pothos provides a status update on our [sysext / composable image-based Linux OS](https://github.com/orgs/flatcar/projects/7/views/16) project

### Release Planning
- https://github.com/orgs/flatcar/projects/7/views/8

### Administrative tasks before meeting starts
- [ ] Create a comment with the list of closed/opened issues/PRs (hint: also sort by time updated)
### Administrative tasks after meeting concluded
- [ ] Create a new Github discussion for the next Flatcar Office Hours: https://github.com/flatcar/Flatcar/discussions/new?category=flatcar-office-hours
- [ ] Close this discussion as "Resolved".

**Comment by miao0miao** [comment]
## Issues Opened
* [#1110](https://github.com/flatcar/Flatcar/issues/1110):  New Package Request: sys-fs/zfs
* [#1109](https://github.com/flatcar/Flatcar/issues/1109):  [RFE] Check for metro capacity before deploying
* [#1107](https://github.com/flatcar/Flatcar/issues/1107):  New Package Request: systemd-journal-remote
* [#1106](https://github.com/flatcar/Flatcar/issues/1106):  Release Flatcar Container Linux Alpha 3654.0.0, Beta 3602.1.2, Stable 3510.2.4
* [#1105](https://github.com/flatcar/Flatcar/issues/1105):  New Package Request: snmpd
* [#1104](https://github.com/flatcar/Flatcar/issues/1104):  [RFE] CAPI Image Publishing Automation
* [#1102](https://github.com/flatcar/Flatcar/issues/1102):  update: patch
* [#1101](https://github.com/flatcar/Flatcar/issues/1101):  update: wget
* [#1100](https://github.com/flatcar/Flatcar/issues/1100):  update: nasm
* [#1099](https://github.com/flatcar/Flatcar/issues/1099):  update: grub
* [#1098](https://github.com/flatcar/Flatcar/issues/1098):  update: re2c
* [#1097](https://github.com/flatcar/Flatcar/issues/1097):  update: nvidia-drivers
* [#1096](https://github.com/flatcar/Flatcar/issues/1096):  Beta 3602.1.0: missing /var/log/sssd directory
* [#1094](https://github.com/flatcar/Flatcar/issues/1094):  Adding TLS mount for EFS - [RFE]
* [#1093](https://github.com/flatcar/Flatcar/issues/1093):  [RFE] Remove torcx from the image build and remove the torcx package from the base OS
* [#1092](https://github.com/flatcar/Flatcar/issues/1092):  [RFE] Build and ship Docker sysext
* [#1091](https://github.com/flatcar/Flatcar/issues/1091):  [RFE] Include Docker packages and dependencies in portage-stable
* [#1090](https://github.com/flatcar/Flatcar/issues/1090):  [RFE] Teach kola to save console logs from the semver machine
* [#1089](https://github.com/flatcar/Flatcar/issues/1089):  update: perl
* [#1088](https://github.com/flatcar/Flatcar/issues/1088):  update: dnsmasq
* [#1087](https://github.com/flatcar/Flatcar/issues/1087):  update: qemu
* [#1086](https://github.com/flatcar/Flatcar/issues/1086):  update: vim
* [#1085](https://github.com/flatcar/Flatcar/issues/1085):  update: shadow
* [#1084](https://github.com/flatcar/Flatcar/issues/1084):  update: libmicrohttpd
* [#1083](https://github.com/flatcar/Flatcar/issues/1083):  update: ipxe
* [#1082](https://github.com/flatcar/Flatcar/issues/1082):  Race condition with libuv fs poll
* [#1081](https://github.com/flatcar/Flatcar/issues/1081):  [RFE] make CI tests of update-ssh-keys work again
* [#1080](https://github.com/flatcar/Flatcar/issues/1080):  update: open-vm-tools
* [#1077](https://github.com/flatcar/Flatcar/issues/1077):  Move nvidia driver installer to /usr partition
* [#1076](https://github.com/flatcar/Flatcar/issues/1076):  are we pointing to the right place for kubernetes downloads?
* [#1075](https://github.com/flatcar/Flatcar/issues/1075):  Release Flatcar Container Linux Alpha 3637.0.0, Beta 3602.1.1, Stable 3510.2.3, LTS 3033.3.14
* [#1074](https://github.com/flatcar/Flatcar/issues/1074):  Tweak kernel config after update
* [#1073](https://github.com/flatcar/Flatcar/issues/1073):  OEM sysext: Fallback download
* [#1072](https://github.com/flatcar/Flatcar/issues/1072):  toolbox instructions do not work as expected

## Issues Closed
* [#1107](https://github.com/flatcar/Flatcar/issues/1107):  New Package Request: systemd-journal-remote
* [#1096](https://github.com/flatcar/Flatcar/issues/1096):  Beta 3602.1.0: missing /var/log/sssd directory
* [#1084](https://github.com/flatcar/Flatcar/issues/1084):  update: libmicrohttpd
* [#1082](https://github.com/flatcar/Flatcar/issues/1082):  Race condition with libuv fs poll
* [#1080](https://github.com/flatcar/Flatcar/issues/1080):  update: open-vm-tools
* [#1077](https://github.com/flatcar/Flatcar/issues/1077):  Move nvidia driver installer to /usr partition
* [#1076](https://github.com/flatcar/Flatcar/issues/1076):  are we pointing to the right place for kubernetes downloads?
* [#1073](https://github.com/flatcar/Flatcar/issues/1073):  OEM sysext: Fallback download
* [#1071](https://github.com/flatcar/Flatcar/issues/1071):  A New Modern Variant of FlatCar.
* [#1069](https://github.com/flatcar/Flatcar/issues/1069):  update: go
* [#1065](https://github.com/flatcar/Flatcar/issues/1065):  mantle: DigitalOcean image removal
* [#1059](https://github.com/flatcar/Flatcar/issues/1059):  Install script not working correctly on the latest versions
* [#1051](https://github.com/flatcar/Flatcar/issues/1051):  Booting issue of Alpha 3619.0.0 for Equinix Metal arm64
* [#1050](https://github.com/flatcar/Flatcar/issues/1050):  update: openssl
* [#1045](https://github.com/flatcar/Flatcar/issues/1045):  update: ncurses
* [#1036](https://github.com/flatcar/Flatcar/issues/1036):  Extend Flatcar quickstart guide
* [#1027](https://github.com/flatcar/Flatcar/issues/1027):  Signature Verification
* [#1025](https://github.com/flatcar/Flatcar/issues/1025):  Parse/decode CrAU protobuf format
* [#999](https://github.com/flatcar/Flatcar/issues/999):  [RFE] Add kubernetes-1.27 tests to the CI
* [#995](https://github.com/flatcar/Flatcar/issues/995):  [RFE] Enable Kernel TLS
* [#734](https://github.com/flatcar/Flatcar/issues/734):  [RFE] Offer Flatcar (and CAPI Flatcar) images via Azure Community Gallery
* [#260](https://github.com/flatcar/Flatcar/issues/260):  When using the suggested config for a laptop cluster, I get 'unit "enable-docker-tcp.service" is enabled, but has no install section so enable does nothing'
